Test/Algorithms/Trainers/LinearSAGTrainer.cpp

```diff
@@ -0,0 +1,191 @@
+//===========================================================================
+/*!
+ * 
+ *
+ * \brief       test case for the Linear SAG-Trainer
+ * 
+ * 
+ * 
+ *
+ * \author      O.Krause
+ * \date        2016
+ *
+ *
+ * \par Copyright 1995-2015 Shark Development Team
```

Comment:
2015 -> 2016
We should really have a yearly cron job somewhere to automate this.